lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -255,6 +282,45 @@ def call(env)
             env[:ui].info(" -- TPM Path:          #{@tpm_path}")
           end
 
+          unless @sysinfo.empty?
+            env[:ui].info(" -- Sysinfo:")
+
+            if @sysinfo.has_key?("bios")
+              env[:ui].info("   -- BIOS:")
+              @sysinfo[:bios].each do |key, value|
+                env[:ui].info("    -> #{key}: #{value}")
+              end
+            end
+
+            if @sysinfo.has_key?("system")
+              env[:ui].info("   -- System:")
+              @sysinfo[:system].each do |key, value|
+                env[:ui].info("    -> #{key}: #{value}")
+              end
+            end
+
+            if @sysinfo.has_key?("base_board")
+              env[:ui].info("   -- Base Board:")
+              @sysinfo[:base_board].each do |key, value|
+                env[:ui].info("    -> #{key}: #{value}")
+              end
+            end
+
+            if @sysinfo.has_key?("chassis")
+              env[:ui].info("   -- Chassis:")
+              @sysinfo[:chassis].each do |key, value|
+                env[:ui].info("    -> #{key}: #{value}")
+              end
+            end
+
+            if @sysinfo.has_key?("oem_strings")
+              env[:ui].info("   -- OEM Strings:")
+              @sysinfo[:oem_strings].each do |value|
+                env[:ui].info("    -> #{value}")
+              end
+            end
+          end
+
```

Comment:
I think this can be condensed a bit:
```
unless @sysinfo.empty?
  env[:ui].info(" -- Sysinfo:")
  sections = [
    [:bios, "BIOS"],
    [:system, "System"],
    [:base_board, "Base Board"],
    [:oem_strings, "OEM Strings"],
  ]

  sections.each do |section_key, description|
    section = @sysinfo.fetch(section_key, [])
    unless section.empty?
      env[:ui].info("   -- #{description}:")
      section.each do |key, value|
        env[:ui].info("    -> #{key}: #{value}")
    end
  end
end
```